src/vivarium_gates_nutrition_optimization/components/intervention.py

```diff
@@ -113,6 +118,19 @@ def update_exposure(self, index, exposure):
             ) * self.ifa_effect_size
 
         return exposure
+    
+    def on_time_step_prepare(self, event):
+        prior_state_pop = self.population_view.get(event.index)
+        prior_state_pop["previous_on_treatment"] = prior_state_pop["on_treatment"]
+        self.population_view.update(prior_state_pop)
+    
+    def on_time_step(self, event):
+        pop = self.population_view.get(event.index)
+        if self.clock() - self.start_date >= timedelta(
+            days=data_values.DURATIONS.INTERVENTION_DELAY_DAYS
+        ):
```

Comment:
This is going to re-set the column's value to `True` on every time after we hit the intervention delay. It is essentially a no-op on every one of these time-steps except the first one. Why not amend this if statement to be true only on that first one?